lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.theme.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * @file 
+ * Preprocess funtions for the dosomething_campaign_run module.
+ */
+
+/**
+ * Implements hook_preprocess_node().
+ */
+function dosomething_campaign_run_preprocess_node(&$vars) {
+  if ($vars['type'] != 'campaign_run' || $vars['view_mode'] != 'full') return;
+
+  $vars['campaign_nid'] = $vars['field_campaigns'][0]['target_id'];
```

Comment:
@angaither I think I need to assign this to this variable in order to pass it to `node--campaign_run.tpl.php`, right?